### PR TITLE
front: fix via modal display

### LINKED ISSUE #7886: Weird alignment in the via modal for long names
### What happened?

Create a path with rolling stock 2TGVDUPL, using the rocket : PLY VSG DN SXA
The alignment "minus" sign of the VSG point is weird 
image
It is ok for shorter names like Dijon : 
image

### What did you expect to happen?

I expected the same alignment

### How can we reproduce it (as minimally and precisely as possible)?

as above

### What operating system, browser and environment are you using?

- **Browser**: Chrome
- **OS**: W11
- **Env**: dev dgexsol

### OSRD version (top right corner `Account` button > `Informations`)

028ae7b

### PR DESCRIPTION
**Pull request by SarahBellaha**
Closes #7886

### CODE REVIEW

**Comment by theocrsb** [comment]
thank you for your PR. behavior is fixed for some names but some still have weird behavior

image

File: front/src/modules/trainschedule/styles/ManageTrainSchedule/_itinerary.scss [hunk 1]
```diff
@@ -58,10 +58,24 @@
       }
 
       &-name {
-        max-width: 47%;
+        width: 45%;
```
Comment (theocrsb) [inline front/src/modules/trainschedule/styles/ManageTrainSchedule/_itinerary.scss:61]: I feel like the behavior is better with 40%

**Review by Uriel-Sautron — APPROVED** [review]
LGTM, tested the behavior is ok

File: front/src/modules/trainschedule/styles/ManageTrainSchedule/_itinerary.scss (hunk 1 above)
Comment (Yohh) [reply front/src/modules/trainschedule/styles/ManageTrainSchedule/_itinerary.scss:61]: that's a UI decision, we should be careful with this kind of suggestions based on personal preferences
also, the via selected is styled by `.reduce-text`, wich has `width: 41%;`

**Comment by Yohh** [comment]
> thank you for your PR. behavior is fixed for some names but some still have weird behavior
> 
> image

nothing like that for me, did your switch work? are you really on the sba/front/fix-via-modal-7886 branch?

**Comment by theocrsb** [comment]
> > thank you for your PR. behavior is fixed for some names but some still have weird behavior
> > image
> 
> nothing like that for me, did your switch work? are you really on the sba/front/fix-via-modal-7886 branch?

My bad. I tried again and it's ok. maybe switch doesn't work.

**Review by theocrsb — APPROVED** [review]
LGTM and tested

**Review by clarani — APPROVED** [review]
LGTM & tested :white_check_mark:

**Pull request merged**